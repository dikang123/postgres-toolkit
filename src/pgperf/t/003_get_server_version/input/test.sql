-- 9.0 to 9.4 is supported.
SELECT pgperf._get_server_version() >= 90;
SELECT pgperf._get_server_version() <= 95;

SELECT pgperf._get_server_version() < 90;
SELECT pgperf._get_server_version() > 95;
