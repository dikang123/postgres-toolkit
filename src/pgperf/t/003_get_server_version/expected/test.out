SELECT pgperf._get_server_version() >= 90;
 ?column? 
----------
 t
(1 row)

SELECT pgperf._get_server_version() <= 95;
 ?column? 
----------
 t
(1 row)

SELECT pgperf._get_server_version() < 90;
 ?column? 
----------
 f
(1 row)

SELECT pgperf._get_server_version() > 95;
 ?column? 
----------
 f
(1 row)

